app/controllers/api/v1/release_packages/pypi/simple_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Api::V1::ReleasePackages
+  class Pypi::SimpleController < Api::V1::BaseController
+    include Rendering
+
+    before_action :scope_to_current_account!
+    before_action :require_active_subscription!
+    before_action :authenticate_with_token
+    before_action :set_product
+
+    authorize :product
+
+    def index
+      artifacts = authorized_scope(apply_scopes(product.release_artifacts)).limit(1_000)
+      authorize! artifacts,
+        with: Products::ReleaseArtifactPolicy
+
+      render 'api/v1/packages/pypi/simple/index',
+        locals: {
+          account: current_account,
+          product:,
+          artifacts:,
+        }
+    end
+
+    private
+
+    attr_reader :product
+
+    def set_product
+      scoped_products = authorized_scope(current_account.products)
+
+      # TODO(ezekg) Add a distribution_engine attribute to product?
```

Comment:
I'm thinking we should add a `distribution_engine` to products, releases and artifacts. That way we can assert we only pull in artifacts that support PyPI, while allowing products to have various engines if needed. We can have releases inherit their engine from its product by default, and artifacts from their release.